app/src/test/kotlin/io/eugenethedev/taigamobile/repositories/SprintsRepositoryTest.kt

```diff
@@ -0,0 +1,192 @@
+            actual = sprintTasks2.map { it.title }
+        )
+    }
+
+    @Test
+    fun `test get sprint issues`() = runBlocking {
+        val sprintIssues1 = sprintsRepository.getSprintIssues(1)
+        val sprintIssues2 = sprintsRepository.getSprintIssues(2) //And were is Issue 3?...
+
+        assertEquals(
+            expected = TestData.projects[0].issues
+                .filter { it.sprint == TestData.projects[0].sprints[0]}
+                .size,
+            actual = sprintIssues1.size
+        )
+        assertEquals(
+            expected = TestData.projects[0].issues
+                .filter { it.sprint == TestData.projects[0].sprints[0]}
+                .map { it.name },
+            actual = sprintIssues1.map { it.title }
+        )
+
+        assertEquals(
+            expected = TestData.projects[0].issues
+                .filter { it.sprint == TestData.projects[0].sprints[1]}
+                .size,
+            actual = sprintIssues2.size
+        )
+        assertEquals(
+            expected = TestData.projects[0].issues
+                .filter { it.sprint == TestData.projects[0].sprints[1]}
+                .map { it.name },
+            actual = sprintIssues2.map { it.title }
+        )
+    }
+
+    @Test
+    fun `test create sprint`() = runBlocking{
+        sprintsRepository.createSprint(
+            "testSprint",
+            LocalDate.of(2000, 1, 1),
+            LocalDate.of(3000, 1, 1)
+        )
+
+        val sprints = sprintsRepository.getSprints(1)
+        assertEquals(
+            expected = "testSprint",
+            actual = sprints.last().name
+        )
+        assertEquals(
+            expected = LocalDate.of(2000, 1, 1),
+            actual = sprints.last().start
+        )
+        assertEquals(
+            expected = LocalDate.of(3000, 1, 1),
+            actual = sprints.last().end
+        )
+    }
+
+    @Test
+    fun `test edit sprint`() = runBlocking {
+        sprintsRepository.editSprint(
+            1,
+            "editSprint",
+            LocalDate.of(2000, 1, 1),
+            LocalDate.of(3000, 1, 1)
+        )
+
+        val sprint = sprintsRepository.getSprint(1)
+        assertEquals(
+            expected = "editSprint",
+            actual = sprint.name
+        )
+        assertEquals(
+            expected = LocalDate.of(2000, 1, 1),
+            actual = sprint.start
+        )
+        assertEquals(
+            expected = LocalDate.of(3000, 1, 1),
+            actual = sprint.end
+        )
+    }
+
+    @Test
+    fun `test delete sprint`() = runBlocking{
+        val nameSprint = sprintsRepository.getSprint(1).name
+        sprintsRepository.deleteSprint(1)
+        val sprints = sprintsRepository.getSprints(1)
+
+        assertEquals(
+            expected = TestData.projects[0].sprints.size - 1,
+            actual = sprints.size
+        )
+        assert(nameSprint !in sprints.map { it.name })
```

Comment:
It is better to use `assert` functions from testing library (`kotlin.test` in our case). So `assertFalse` or `assertTrue` are more suitable here